vivisect/symboliks/archs/i386.py

```diff
@@ -1471,6 +1471,15 @@ class MsFastCall(i386SymCallingConv, e_i386.MsFastCall):
 class BFastCall(i386SymCallingConv, e_i386.BFastCall):
     pass
 
+class ThisCall_Caller(i386SymCallingConv, e_i386.ThisCall_Caller):
```

Comment:
are these really just copies with a different name?  
there should be some function (both sym and emu versions) which mathematically perform/show the differences.

Reply:
They're not particularly copies. They're analogs to the analogs. They're analogs to the caller cleanup version in the envi modules. The reason we need these ones is so that deallocateCallSpace returns the right values when it's called by setSymbolikReturn.

But you are right in that it is a good idea to add a test that demonstrates the difference. So I will do that before merging.

Reply:
thanks.  that makes more sense.